hbase_sql/src/main/java/com/splicemachine/mrio/api/core/AbstractSMInputFormat.java

```diff
@@ -89,6 +90,7 @@
         Partition clientPartition = new ClientPartition(connection, table.getName(), table, clock, driver.getPartitionInfoCache());
         int retryCounter = 0;
         boolean refresh = false;
+        this.splits = conf.getInt(MRConstants.SPLICE_SPLITS_PER_TABLE, 0);
```

Comment:
should the default be 0?

Reply:
0 means use the usual splitBlockSize heuristic